scripts/linux/trcE2ETest.sh

```diff
@@ -220,8 +219,8 @@ function prepare_test_from_artifacts() {
     fi
 
     if [[ "${TEST_NAME,,}" == "${CONNECTIVITY_TEST_NAME,,}" ]]; then
+        sed -i -e "s@<TestDuration>@$TEST_DURATION@g" "$deployment_working_file"
```

Comment:
As a preventive measure, I think it would be a good idea to have the shell script check if the parameter is accidentally passed in here. If it is in the wrong "TestMode" it would yell at you.

Reply:
I agree, and we should also do so for all the test-specific vars. I'd rather scope this PR to just fixing the twin tests though. My goal is to see if nested twin tests have any serious issues before we release.

Reply:
I looked into this more and I think we should have a second PR where we get rid of TEST_MODE, use TEST_NAME instead. Then in the same PR we can add these checks based on TEST_NAME.